docs/source/getting_started/PackageSummary.rst

```diff
@@ -341,3 +341,54 @@ Supported Platforms and Packages
 -  Requires Python 3.8 or later
 -  See `requirements.txt <https://github.com/microsoft/msticpy/blob/master/requirements.txt>`__
    for more details and version requirements.
+
+
+
+aiagents Module: RAG Agent
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:py:mod:`msticpy.aiagents.ragagent`
+
+The **aiagents** module in MSTICpy introduces the **Retrieval-Augmented Generation (RAG) Agent**.
+This agent is designed for Q&A about MSTICpy. It enhances InfoSec investigations and
+threat hunting in Jupyter Notebooks by providing contextually relevant responses using large language
+models and external knowledge retrieval from the MSTICpy documentation.
+
```

Comment:
maybe have a reference to the config docs,

Reply:
This is addressed now on line 391.